src/main/java/hw3/ex2/components/HeaderMenu.java

```diff
@@ -0,0 +1,52 @@
+package hw3.ex2.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HeaderMenu {
+
+    private WebDriver driver;
+
+    @FindBy(xpath = "//ul[@class='uui-navigation nav navbar-nav m-l8']")
```

Comment:
There is not necessarily has all list of the classes in the locator. `//ul[@class='uui-navigation nav navbar-nav m-l8']` please choose one which more prefarable

Reply:
Solved: now I'm using `@FindBy(className = "navbar-nav")`